src/io/jesi/backpack/collection.cljc

```diff
@@ -279,3 +279,21 @@
          (assoc-non-empty :same @same)
          (assoc-non-empty :removed vec @removed)
          persistent!))))
+
+(defn sorted?
+  "True if a collection is sorted by means of a 2 or 3 way comparator"
+  [comp coll]
+  (let [coll (seq coll)]
+    (cond
+      (nil? coll)
+      true
+
+      (= 1 (count coll))
```

Comment:
Combine cond by:
```suggestion
      (< (count coll) 2)
```
since `(count nil)` is `0`

Also, instead of `cond` use `or`